### PR TITLE
stale: process older PRs first

### PR DESCRIPTION
**Pull request by ericLemanissier**
---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -60,4 +62,4 @@ jobs:
             recent activity. Thank you for your contributions.
 
         # Max number of operations per run
-        operations-per-run: 30
+        operations-per-run: 100
```
Comment (uilianries) [inline .github/workflows/stale.yml:65]: Let's keep 30. We are not in a hurry with stale PRs.

Comment (ericLemanissier) [reply .github/workflows/stale.yml:65]: As you which. 
For the record, with the current state of the stale action, 2 operations are consumed to check the last modification date of all PRs having the stale label. This means conan-center-index will never have more than 15 stale PRs

Comment (uilianries) [reply .github/workflows/stale.yml:65]: No problem, we still can re-visit a bunch of them. Thank you about this information. 

About the rate-limit, we have been doing some experiments using Github API (internally), so as less as we use in actions better will be.

**Review by uilianries — APPROVED** [review]
LGTM

**Review by jcar87 — APPROVED** [review]

**Pull request merged**